recipes/libgeotiff/all/patches/fix-cmake.patch

```diff
@@ -0,0 +1,124 @@
+--- a/libgeotiff/CMakeLists.txt
++++ b/libgeotiff/CMakeLists.txt
+@@ -80,7 +80,9 @@ SET(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib)
+ 
+ IF(WIN32)
+     IF(MSVC)
+-        ADD_DEFINITIONS(-DBUILD_AS_DLL=1)
++        IF(BUILD_SHARED_LIBS)
++            ADD_DEFINITIONS(-DBUILD_AS_DLL=1)
++        ENDIF()
+         ADD_DEFINITIONS(/DW4)
+         if (NOT (MSVC_VERSION VERSION_LESS 1400))
+             ADD_DEFINITIONS(-D_CRT_SECURE_NO_DEPRECATE)
+@@ -92,7 +94,7 @@ IF(WIN32)
+ ENDIF()
+ 
+ IF(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX)
+-    SET(COMPILE_FLAGS "-fPIC -Wall -Wno-long-long")
++    SET(COMPILE_FLAGS "-Wall -Wno-long-long")
+     SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${COMPILE_FLAGS} -std=c99")
+     SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${COMPILE_FLAGS} -std=c++98")
+     IF(GEOTIFF_BUILD_PEDANTIC)
+@@ -119,80 +121,6 @@ SET(WITH_UTILITIES TRUE CACHE BOOL "Choose if GeoTIFF utilities should be built"
+ INCLUDE_DIRECTORIES(${PROJECT_SOURCE_DIR})
+ INCLUDE_DIRECTORIES(${PROJECT_SOURCE_DIR}/libxtiff)
+ 
+-# TIFF support - required, default=ON
+-SET(WITH_TIFF TRUE CACHE BOOL "Choose if TIFF support should be built")
+-
+-FIND_PACKAGE(PROJ NO_MODULE QUIET)
+-if (NOT PROJ_FOUND)
+-    FIND_PACKAGE(PROJ)
+-endif ()
+-
+-IF(PROJ_FOUND)
+-    INCLUDE_DIRECTORIES(${PROJ_INCLUDE_DIR})
+-ELSE()
+-    MESSAGE(FATAL_ERROR "Failed to detect PROJ >= 6")
+-ENDIF()
+-
+-# Zlib support - optional, default=OFF
+-SET(WITH_ZLIB FALSE CACHE BOOL "Choose if zlib support should be built")
+-
+-IF(WITH_ZLIB)
```

Comment:
Why don't you need zlib and jpeg anymore? Are they required by libtiff only?

Reply:
As I said in my first comment, zlib and libjpeg aren't direct dependencies. libgeotiff tries to manage manually dependencies of libtiff.

Reply:
sorry, I missed it (almost 1 month ago). So I think we are good to go.